src/helpers/DetailedResultsHelpers.ts

```diff
@@ -100,3 +100,30 @@ export function addIdsToPopulationResult(populationResult: PopulationResult, pop
     }
   }
 }
+
+/**
+ * Finds the measure observation that reference the desired population in their criteria reference. If one exists,
+ *  sets the result to false and the observations to null
+ *
+ *  NOTE: the usage of criteriaReference to identify a measure observation only really applies for Ratio measures
+ *  where observations can be done on both the numerator and the denominator. For logic relating to CV measures that have a measure-population,
+ *  the nulling of irrelevant observations happens already via `handlePopulationValues`
+ */
+export function nullCriteriaRefMeasureObs(
+  group: fhir4.MeasureGroup,
+  populationResults: PopulationResult[],
+  desiredPopulationType: PopulationType
+): void {
+  const popResults = populationResults.filter(result => result.populationType === PopulationType.OBSERV);
+  if (desiredPopulationType === PopulationType.NUMER && popResults.length === 1) {
+    // If only one measure-observation population, we know it relates to numerator
```

Comment:
I think technically it could yeah. The safer move is just to null out the result regardless of what the desired populationType is. We should never really even be in this case in actual examples